models/hospital.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Hospital extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  Hospital.init(
+    {
+      hospital_uuid: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        allowNull: false,
+      },
+      hospitalname: { type: DataTypes.STRING.BINARY, allowNull: false },
```

Comment:
Per the migration schema change you committed earlier, this can now just be a DataTypes.STRING

Reply:
if we want to have more JavaScript-y naming convention (i.e. camelCase) for attributes, you can do that in the init by explicitly specifying it in the field attribute, ie:

```
hospitalName: {
  ...
  field: 'hospitalname'
},
```

Reply:
Oh nice, thanks for the tip. I'm realizing that Sequelize is pretty customizable.